layouts/partials/publicationsSummary.html

```diff
@@ -7,6 +7,6 @@ <h3 class="mb-0">{{ .Params.pubtype }} - <a href="{{ .Permalink }}">{{ .Params.t
   </div>
   <div class="resume-date text-md-right">
     <span class="text-primary">{{ .Params.date | dateFormat "January 2006" }}</span>
-    <img src="{{ .Params.image }}" style="max-width:250px;padding:5px;"/>
+    <img src="{{ .Params.image }}" {{if .Params.imageDescription }}alt="{{ .Params.imageDescription }}"{{ else }}alt=""{{ end }} style="max-width:250px;padding:5px;"/>
```

Comment:
why is an empty alt preffered over just omitting it?

Reply:
Empty Alt can be used to indicate an image is decorative, see https://www.w3.org/WAI/tutorials/images/decorative/